docs/documentation/platform/dynamic-secrets/ldap.mdx

```diff
@@ -0,0 +1,164 @@
+    </ParamField>
+
+    <ParamField path="Max TTL" type="string" required>
+    	Maximum time-to-live for a generated secret.
+    </ParamField>
+
+    <ParamField path="URL" type="string" required>
+    	LDAP url to connect to. _(Example: ldap://your-ldap-ip:389 or ldaps://domain:636)_
+    </ParamField>
+
+    <ParamField path="BIND DN" type="string" required>
+        DN to bind to. This should have permissions to create a new users.
+    </ParamField>
+
+    <ParamField path="BIND Password" type="string" required>
+        Password for the given DN.
+    </ParamField>
+
+    <ParamField path="CA" type="text">
+    	CA certificate to use for TLS in case of a secure connection.
+    </ParamField>
+
+    <ParamField path="Creation LDIF" type="text" required>
+    	LDIF file to run while creating a user in LDAP. This can include extra steps to assign the user to groups or set permissions.
+        Here `{{Username}}`, `{{Password}}` and `{{EncodedPassword}}` are templatized variables for the username and password generated by the dynamic secret.
+
+        **OpenLDAP** Example:
+        ```
+        dn: uid={{Username}},dc=acme,dc=com
+        changetype: add
+        objectClass: top
+        objectClass: person
+        objectClass: organizationalPerson
+        objectClass: inetOrgPerson
+        cn: John Doe
+        sn: Doe
+        uid: jdoe
+        mail: jdoe@jumpcloud.com
+        userPassword: {{Password}}
+        ```
+
+        **Active Directory** Example:
+        ```
+        dn: CN={{Username}},OU=Test Create,DC=mcs19,DC=tech
+        changetype: add
+        objectClass: top
+        objectClass: person
+        objectClass: organizationalPerson
+        objectClass: user
+        userPrincipalName: {{Username}}@mcs19.tech
+        sAMAccountName: {{Username}}
+        unicodePwd::{{EncodedPassword}}
+        userAccountControl: 66048
+
+        dn: CN=test-group,OU=Test Create,DC=mcs19,DC=tech
+        changetype: modify
+        add: member
+        member:  CN={{Username}},OU=Test Create,DC=mcs19,DC=tech
+        -
+        ```
+    </ParamField>
+
+    <ParamField path="Revocation LDIF" type="text" required>
+    	LDIF file to run while revoking a user in LDAP. This can include extra steps to remove the user from groups or set permissions.
+        Here `{{Username}}` is a templatized variable for the username generated by the dynamic secret.
+
+        **OpenLDAP** Example:
+        ```
+        dn: uid={{Username}},dc=acme,dc=com
+        changetype: delete
+        ```
+
+        **Active Directory** Example:
+        ```
+        dn: CN={{Username}},OU=Test Create,DC=mcs19,DC=tech
+        changetype: delete
+        ```
+    </ParamField>
+
+    <ParamField path="Rollback LDIF" type="text">
+    	LDIF file to run incase create fails midway.
+        Here `{{Username}}`, `{{Password}}` and `{{EncodedPassword}}` are templatized variables for the username generated by the dynamic secret.
+    </ParamField>
+
```

Comment:
this part seems to be empty?
<img width="912" alt="image" src="https://github.com/user-attachments/assets/5cb1d4af-bb95-4d32-b6d6-8467ea2b02b0">

Reply:
I thought the example isn't needed here because its very usecase specific, but I will just make one.